test/test_cohort_defaults.py

```diff
@@ -32,9 +32,9 @@
 
 FILE_FORMAT_1 = "patient_format1_%s.vcf"
 
-def test_default_filter_fn():
+def test_default_filter_fn(summary_fn=variant_count):
```

Comment:
I asked to not duplicate code, that's right. I'm just asking if you need another function called `test_default_filter_fn_variant_count` that calls this for it to run.